lib/rubocop/directive_comment.rb

```diff
@@ -53,5 +48,31 @@ def range
     def match_captures
       @match_captures ||= comment.text.match(DIRECTIVE_COMMENT_REGEXP)&.captures
     end
+
+    def disabled?
+      return @disabled if defined? @disabled
+
+      @disabled = %w[disable todo].include?(mode)
+    end
+
+    def mentioned_all?
```

Comment:
This doesn't seem like an optional name to me. Perhaps `all_cops?` would be better or something along those lines.

Reply:
Replaced